### PR TITLE
Add Fastlane metadata for F-Droid

### PR DESCRIPTION
**Pull request by rollerozxa**
This PR adds Fastlane metadata for F-Droid. Just the basics for 5.10.0, consisting of the title, short description, long description and the icon.

I'm not fully happy with the full description, but I basically tried to rewrite what the old description was on F-Droid (https://f-droid.org/en/packages/net.minetest.minetest/) while trying to emphasise the engine/platform aspect of Luanti. Someone should write something better that can also be used in e.g. the Linux metainfo description and other places where we could use a good description of what Luanti really is, but I hope this will do.

## To do
This PR is Ready for Review.

## How to test
Read

### CODE REVIEW

**Comment by grorp** [comment]
>  Someone should write something better that can also be used in e.g. the Linux metainfo description and other places where we could use a good description of what Luanti really is, but I hope this will do.

What you wrote is at least better than Linux metainfo one. But indeed, we really need someone to write a proper engine-y/platform-y long description that can be used everywhere.

File: fastlane/metadata/android/en-US/full_description.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Luanti is a voxel-based game platform with multiplayer capabilities.
+
+The objective in the games available varies, from sandbox games where you can explore, dig and build in a very large voxel world, to arcade style games made with the engine.
+
+There are a wide range of built-in map generators for procedually generated terrain, and voxel-based lighting that adjusts after the environment and light sources.
+
+All games and mods are written in Lua using the scripting API which is easy to use and get started with for modders.
+
+Luanti is available for Windows, macOS, GNU/Linux, FreeBSD, OpenBSD, DragonFly BSD, and Android.
```
[the remark this answers is not in the document]
Comment (grorp) [reply fastlane/metadata/android/en-US/full_description.txt:9]: also the "servers" aspect is too small here imo

**Review by grorp — APPROVED** [review]
I'll trust you that the structure is correct

File: fastlane/metadata/android/en-US/full_description.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (appgurueu) [reply fastlane/metadata/android/en-US/full_description.txt:9]: :+1: on grorp's suggestions

The focus of the description generally seems a bit arbitrary to me ..? For example the mention of three different BSD versions (which I'm not even so sure work terribly well, the ports are not done by the core team) while just summarizing Linux distros as "GNU/Linux", or the mention of the "voxel-based lighting system".

I would say that things like multiplayer, possibility of PvP, mobs etc. should be emphasized more.

That said I don't want to bikeshed this to death. This description is still a significant improvement and can be improved further in the future.

[the remark this answers is not in the document]
Comment (sfan5) [reply fastlane/metadata/android/en-US/full_description.txt:9]: we should drop the "Luanti is available for" sentence, it's not relevant for Android at all.

[the remark this answers is not in the document]
Comment (SmallJoker) [reply fastlane/metadata/android/en-US/full_description.txt:9]: :+1: on grorp's suggestions. The last sentence is not really needed, and technically incorrect for GNU/Linux. Luanti also runs on other libc implementations.

[the remark this answers is not in the document]
Comment (sfan5) [reply fastlane/metadata/android/en-US/full_description.txt:9]: (copying to apply)
suggestions:

```suggestion
Luanti is a voxel-based game platform with multiplayer capabilities.

The objective of the available games varies, from sandbox games where you can explore, dig and build in a very large voxel world, to arcade-style games.

There is a wide range of built-in map generators for procedurally generated terrain, and voxel-based lighting that adapts to the environment and light sources.

All games and mods are written in Lua with a scripting API that is easy to use and makes it easy for modders to get started.
```

**Review by sfan5 — APPROVED** [review]

**Pull request merged**